diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2a3949a92..968959599 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,8 +2,8 @@ cmake_minimum_required(VERSION 3.12)
 
 project(rosidl_generator_type_description)
 
-find_package(ament_cmake_python REQUIRED)
-find_package(ament_cmake_ros REQUIRED)
+find_package(ament_cmake REQUIRED)
+# find_package(ament_cmake_ros REQUIRED)
 
 ament_index_register_resource("rosidl_generator_packages")
 ament_python_install_package(${PROJECT_NAME})
